examples/server.js

```diff
@@ -134,4 +133,8 @@ server = http.createServer(function (req, res) {
   }
 });
 
-server.listen(argv.p || 3000);
+const PORT = argv.p || 3000;
+
+server.listen(PORT);
+
+console.log("Examples running on " + PORT);
```

Comment:
Had to read the source code to figure this out. Adding a console log for future devs